content/people/catherine-kretzschmar.md

```diff
@@ -1,6 +1,6 @@
 ---
-avatar: /images/people/catherine-kretzschmar.jpg
-avatar_small: /images/people/catherine-kretzschmar_small.jpg
+avatar: /images/people/cat.jpg
+avatar_small: /images/people/cat_small.jpg
```

Comment:
For the sake of consistency I would rename `cat.jpg` and `cat_small.jpg` to `catherine-kretzschmar.jpg` and `catherine-kretzschmar_small.jpg` just so they remain consistent with the username

Reply:
That is an excellent point, will do.